### PR TITLE
Initial `pyth aptos` CLI

### PR DESCRIPTION
**Pull request by tompntn**
This is a simple CLI utility, designed to abstract some simple atomic operations on the Pyth Target Chain Aptos contracts.

### CODE REVIEW

File: aptos/cli/src/commands/aptos.ts [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+      "Deploy a package",
+      (yargs) => {
+        return yargs
+          .positional("package-dir", { type: "string" })
+          .positional("account", { type: "string" })
+          .option("named-addresses", { type: "string" })
+          .option("network", network);
+      },
+      async (argv) => {
+        const artefact = serializePackage(
```
Comment (ali-bahjati) [inline aptos/cli/src/commands/aptos.ts:91]: is there a specific reason it is `artefact`? :D

**Review by ali-bahjati — APPROVED** [review]
Looks great to me!

I don't understand everything here, so I do not understand all the commands. I assume the wormhole contract command is useful for tilt setup 🤔 Maybe a description can be helpful.

Also in some places <package-dir> is passed, but maybe if it is only pyth package, hard-coding it as config make the cli easier. But of course it could be used for wormhole for tilt as well.

File: aptos/cli/src/commands/aptos.ts (hunk 1 above)
Comment (tompntn) [reply aptos/cli/src/commands/aptos.ts:91]: I took inspiration from the phrase "build artefact", but better name suggestions are welcome 😄

**Review by jayantk — APPROVED** [review]
the code itself seems fine,though i don't really understand what it's supposed to do. would suggest adding a README explaining how to use this utility (in a separate PR).

**Pull request merged**